content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_SQL_Server_Deployment_Guide--Enabling_Always_Encrypted_with_YubiHSM_2/Getting_Help_and_Further_Reading.adoc

```diff
@@ -0,0 +1,28 @@
+== Getting Help and Further Reading
+
+Should you require assistance when using this guide to deploy YubiHSM 2 with Microsoft SQL Server Always Encrypted, start by referencing the product documentation and currently known issues:
+
+link:../../developers.yubico.com/[Yubico Developers website]
+
+link:https://support.yubico.com/support/home[Yubico Support]
+
+link:../../developers.yubico.com/YubiHSM2/[Product documentation]
```

Comment:
this link is broken, it should probably be just ../../

Reply:
Appreciate your diplomacy. It didn't even point to "product documentation"; I've fixed it so it points to "YubiHSM 2 page".